app/views/books/delete.html.erb

```diff
@@ -3,6 +3,9 @@
 <p>Are you sure you want to delete this book?</p>
 <%= form_for(@book, method: "delete") do |f| %>
```

Comment:
code violation: use meaningful variables. It seems that you have made the variable as f and it could've been something more meaningful like form